### PR TITLE
CI: download newest OOSDK toolchain

### PR DESCRIPTION
**Pull request by illusionyy**

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -35,17 +35,9 @@ jobs:
...
-      uses: actions/cache@v3
-      with:
-        path: ${{ env.OO_PS4_TOOLCHAIN }}
-        key: ${{ runner.os }}-oosdk-llvm-${{ env.llvm_ver }}-${{ env.OO_SDK_URL }}
-
     # temporary release until 0.53 is released
     - name: Download OpenOrbis Toolchain
-      if: steps.cache-oosdk.outputs.cache-hit != 'true'
-      run: curl -sL ${{ env.OO_SDK_URL }} | tar xz -C ./
+      run: curl -sL https://github.com/illusion0001/OpenOrbis-PS4-Toolchain/releases/latest/download/toolchain.tar.gz | tar xz -C ./
```
Comment (bucanero) [inline .github/workflows/build.yml]: Shouldn't we just update the URL on `env.OO_SDK_URL` ?

(or remove the OO_SDK_URL env variable)

**Review by bucanero — CHANGES_REQUESTED** [review]

File: .github/workflows/build.yml (hunk 1 above)
Comment (illusionyy) [reply .github/workflows/build.yml]: It was only used for caching key, it does not make a difference in terms of download speed in my testing so I just removed it. Roughly 2-3 seconds to download and extract it using cache and without

**Review by bucanero — APPROVED** [review]

**Comment by bucanero** [comment]
I think we should do a quick test and confirm if the generated `.pkg` works as expected, to avoid problems showing up later  due to the updated SDK

**Comment by illusionyy** [comment]
tested, app still works same as before

**Pull request merged**